tslint.json

```diff
@@ -48,7 +45,6 @@
     "no-switch-case-fall-through": true,
     "no-trailing-whitespace": true,
     "no-unused-expression": true,
-    "no-unused-variable": true,
```

Comment:
Please re-add this rule

Reply:
This rule is deprecated and is set in the compilerOptions of tsconfig.json.